### PR TITLE
list and change status orders

### PR DESCRIPTION
**Pull request by kien2572001**
## Related Tickets
- ticket redmine
- https://edu-redmine.sun-asterisk.vn/issues/67363
- https://edu-redmine.sun-asterisk.vn/issues/67362
- https://edu-redmine.sun-asterisk.vn/issues/67361

## WHAT (optional)
- Xây dựng trang quản lí order 
- Chức năng đồng ý hoặc từ chối đơn hàng
- Chức năng tìm kiếm đơn hàng

## HOW
- Sử dụng ransack form để tìm kiếm

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2023-09-14 23-26-24

### CODE REVIEW

**Comment by kien2572001** [comment]
ready

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Admin::OrdersController < ApplicationController
+  layout "admin"
+
+  def index
+    @q = Order.ransack(params[:q])
+    @pagy, @orders = pagy(@q.result(distinct: true).newest)
+  end
+
+  def update
+    @order = Order.find_by id: params[:id]
```
Comment (vanvtt-0952) [inline app/controllers/admin/orders_controller.rb]: ... tự hỉu nha

File: app/helpers/admin/orders_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module Admin::OrdersHelper
```
Comment (vanvtt-0952) [inline app/helpers/admin/orders_helper.rb]: Xóa file

File: app/views/admin/shared/_order.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <% when "rejected" %>
+    <span class="badge text-bg-danger text-white"><%= t(".rejected") %></span>
+  <% else %>
+    <span class="badge badge text-bg-dark text-white"><%= t(".unknown") %></span>
+  <% end %>
+</td>
+<td><%= t(".details") %></td>
+<td class="operations">
+  <div class="action_buttons">
+    <% if order.status == "pending" %>
```
Comment (vanvtt-0952) [inline app/views/admin/shared/_order.html.erb]:
```suggestion
    <% if order.pending? %>
```

**Comment by kien2572001** [comment]
ready

**Pull request merged**